Gemfile

```diff
@@ -22,6 +22,6 @@ group :test do
   gem 'rubocop-performance', require: false
 
   platform :mri, :truffleruby do
-    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'master'
+    gem 'liquid-c', github: 'Shopify/liquid-c', ref: 'pz-block-body-buffer'
```

Comment:
It doesn't seem like this pull request should depend on the liquid-c one.  It seems like that creates a circular dependency between the two pull requests